system/Session/Handlers/RedisHandler.php

```diff
@@ -142,17 +142,19 @@ protected function setSavePath(): void
             }
         }
 
-        $password = $query['auth'] ?? null;
-        $database = isset($query['database']) ? (int) $query['database'] : 0;
-        $timeout  = isset($query['timeout']) ? (float) $query['timeout'] : 0.0;
-        $prefix   = $query['prefix'] ?? null;
+        $persistent = filter_var($query['persistent'], FILTER_VALIDATE_BOOLEAN) ? (bool) $query['persistent'] : null;
```

Comment:
Starting PHP 8.0, we can use FILTER_VALIDATE_BOOL instead.

```suggestion
        $persistent = filter_var($query['persistent'], FILTER_VALIDATE_BOOL) ? (bool) $query['persistent'] : null;
```